From fccc7baac8739ca3e516ea83780809ecb989b643 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Mon, 5 Sep 2022 08:17:28 +0300
Subject: [PATCH] chromium: update to 105.0.5195.102

Upstream-Status: Pending

Signed-off-by: Maksim Sisov <msisov@igalia.com>

---
 tools/polymer/html_minifier.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/polymer/html_minifier.js b/tools/polymer/html_minifier.js
index e86ef11a66..d2f9b1352d 100644
--- a/tools/polymer/html_minifier.js
+++ b/tools/polymer/html_minifier.js
@@ -14,7 +14,7 @@ const minify =
         .minify;
 
 const path = require('path');
-const fs = require('fs/promises');
+const fs = require('fs').promises;
 
 // Regex to extract the CSS contents out of the HTML string. It matches anything
 // that is wrapped by a '<style>...</style>' pair.
